ruby-tapas-downloader.rb

```diff
@@ -35,12 +35,15 @@ def get_episode_links
   def download_missing_episodes
     yield if block_given?
 
+    browser = LoggedBrowser.new.call
+
     get_episode_links.each do |link|
       href = link.href
       log_status_update(href)
 
-      EpisodeLink.new(href).call
+      EpisodeLink.new(href).call(browser)
     end
+    browser.close
```

Comment:
I think right here using ensure would be best, that way if we end up erroring it will always teardown.